.github/workflows/build-windows.yml

```diff
@@ -141,6 +127,17 @@ jobs:
           submodules: true
           fetch-depth: 0
 
+      - name: Setup MSVC toolset
+        uses: TheMrMilchmann/setup-msvc-dev@v3
+        with:
+          arch: x64
+          # By default Visual Studio 2022 chooses the earliest installed toolset
+          # version for the main build and vcpkg chooses the latest. Force it to
```

Comment:
Is there an issue about this that we can cross-reference inline when someone tries to understand this in 6 mo?

Reply:
I could not find anything related to Visual Studio's default behavior; the issues I found were about MSBuild specifically.